adopt-a-dog/dogs.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+    <meta charset="utf-8">
+    <title>ADOPTADOG</title>
+</head>
+<body>
+    <header>
+      <img src="adoptadog-logo.svg" alt="site-logo" width=300>
+      <h1>Seattle's premiere dog adoption agency</h1>
+      <img src="cart-icon.svg" alt="shopping cart icon" width=30>
+      <form action="#" method="post">
+        <input type="button" name="checkout" value="Checkout">
+      </form>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h2>Here are all of our Dogs!</h2>
+      <article class="dog-profile">
+        <img src="images/alex-lacayo-756508-unsplash.jpg" alt="photo of Murphy" width=150>
+        <h3>Murphy</h3>
+        <p>Cost to Adopt: $123.45</p>
+        <p>Corrum volorit iandae
+nimaxim cum restia volor
```

Comment:
More indents...if they're wrong in your editor, make sure to get them nice and indented...